### PR TITLE
Update user interface

### PR DESCRIPTION
**Pull request by thinkh**
**prerequisites**: 
 * [x] branch is up-to-date with the branch to be merged with, i.e. develop
 * [x] build is successful
 * [x] code is cleaned up and formatted 

### Summary
 
* Change header and footer to grey background-color 
* Decrease footer min-height and change footer text
* Add proper labels to columns with multiple words 
* Make download button smaller 
* Remove space to checkbox in side panel since it is defined in library
* Darker label text color
* Modify radio button style 
* create explicit matrix model + toolbar action 
* Update style for grouping and sorting hierarchies

### CODE REVIEW

File: src/export.ts [hunk 1]
```diff
@@ -20,12 +20,12 @@ export default function initExport() {
     evt.preventDefault();
     evt.stopPropagation();
     const ranking = shared.lineup!.data.getRankings()[0];
-    const cols = ranking.flatColumns.filter((d) => !isSupportType(d));
+    const cols = ranking.flatColumns.filter((d: any) => !isSupportType(d));
```
Comment (sgratzl) [inline src/export.ts]: why are these casts needed?

File: src/index.html [hunk 2]
```diff
@@ -12,27 +12,31 @@
   <!--Let browser know website is optimized for mobile-->
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-  <meta name="description" content="LineUp is an interactive technique designed to create, visualize and explore rankings of items based on a set of heterogeneous attributes"/>
-  <meta property="og:image:height" content="247"/>
-  <meta property="og:image:width" content="500"/>
-  <meta property="og:title" content="LineUp App"/>
+  <meta name="description" content="LineUp is an interactive technique designed to create, visualize and explore rankings of items based on a set of heterogeneous attributes"
```
Comment (sgratzl) [inline src/index.html]: why the formatting?

**Review by sgratzl — COMMENTED** [review]
in general, I'm not a fan of the gray setup. it doesn't match at all the website anymore. 

Moreover, I actually don't want the matrix column as part of this app, since the feature is unstable and hacky

File: src/export.ts (hunk 1 above)
Comment (thinkh) [reply src/export.ts]: Due to compiler errors.

**Comment by thinkh** [comment]
Can we change the color of the lineup homepage as well, so that it matches the demo style?

Should I extract the matrix splitting?

File: src/index.html (hunk 2 above)
Comment (thinkh) [reply src/index.html]: Due to VSC formatter.

File: src/export.ts (hunk 1 above)
Comment (sgratzl) [reply src/export.ts]: if there are compile errors, why does the circleci works for the latest master build?

Comment (thinkh) [reply src/export.ts]: Fixed.

File: src/index.html (hunk 2 above)
Comment (thinkh) [reply src/index.html]: Fixed.

**Comment by sgratzl** [comment]
> Can we change the color of the lineup homepage as well, so that it matches the demo style?

I don't like and want a pure gray website. It looks so bleak.

**Comment by sgratzl** [comment]
> Should I extract the matrix splitting?

yes, please remove it from this PR

**Comment by thinkh** [comment]
I close this PR, because I created two new PRs (#23 and #24) and left out the matrix stratification.

**Pull request closed without merging**